packages/core/src/components/TextArea/index.tsx

```diff
@@ -30,9 +32,11 @@ export default class TextArea extends React.Component<Props, State> {
   };
 
   static defaultProps = {
+    alwaysHighlight: false,
```

Comment:
I'm not a fan of adding these extra props to `TextArea`. Add a new `proofreaderProps` prop that is an object of props to pass down.